brage-import/src/main/java/no/sikt/nva/brage/migration/model/PublicationRepresentation.java

```diff
@@ -0,0 +1,8 @@
+package no.sikt.nva.brage.migration.model;
+
+import no.sikt.nva.brage.migration.record.Record;
+import no.unit.nva.model.Publication;
+
+public record PublicationRepresentation(Record brageRecord, Publication nvaPublication) {
```

Comment:
This class was created in order to have a secure source for the Brage-handle. I  think creating reports of additional identifiers is a terrible idea.